### DISCUSSION TITLE
Why is fps so low during studio preview?

### DISCUSSION THREAD

**Discussion by winner106** [remotion-dev/remotion#3261] [Q&A]
I set the fps to 60, and the rendering effect is fine, but why is the fps in the studio preview so low?
![image](https://github.com/remotion-dev/remotion/assets/50519642/e0403535-ea44-4659-8af2-d4ae463d2b0b)

**Comment by JonnyBurger** [comment]
Do you have set the playbackrate to 0.25x by any chance?
Other than that, I have never seen this issue appear. Need some more information if it keeps happening.
